ark/dist/chart/templates/crd/ark.mckinsey.com_tools.yaml

```diff
@@ -37,6 +37,8 @@ spec:
             type: object
           spec:
             properties:
+              agent:
```

Comment:
can we do this:

```
agent:
  name:
```

like this: https://github.com/McK-Internal/agents-at-scale-marketplace/pull/44/files#diff-709d13403e41ef38acebb22d149c6c1f52d0f31d3df371fb11c86d6192e29f11R18
 
cause later we may need to add 'namespace' etc